redash/query_runner/sqlite.py

```diff
@@ -81,12 +78,6 @@ def run_query(self, query, user):
             connection.cancel()
             error = "Query cancelled by user."
             json_data = None
-        except Exception as e:
-            # handle unicode error message
-            err_class = sys.exc_info()[1].__class__
-            err_args = [arg.decode('utf-8') for arg in sys.exc_info()[1].args]
-            unicode_err = err_class(*err_args)
-            reraise(unicode_err, None, sys.exc_info()[2])
```

Comment:
Previously this was catching all errors, even if they weren't UnicodeErrors, so I would suggest to keep this.

Reply:
I think this was not catching `UnicodeError` but rather catching everything, decoding content and reraising. I still think that this is not needed on Python 3.

Reply:
I agree w/ @NicolasLM , this seems unnecessary anymore (it was supposed to handle error message *with* Unicode).